// [[Reason: Sourcemap is more detailed]] ////

//// [tests/cases/compiler/declarationEmitUsingTypeAlias1.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,6 +1,6 @@
 
 //// [src/index.d.ts.map]
-{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["index.ts"],"names":[],"mappings":"AAAA,OAAO,EAAE,QAAQ,EAAE,MAAM,UAAU,CAAC;AAEpC,eAAO,MAAM,GAAG,UAAW,QAAQ,KAAG,QAErC,CAAC;AAEF,eAAO,MAAM,GAAG,UAAW,QAAQ,UAElC,CAAC"}
+{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["index.ts"],"names":[],"mappings":"AAAA,OAAO,EAAE,QAAQ,EAAE,MAAM,UAAU,CAAC;AAEpC,eAAO,MAAM,GAAG,GAAI,KAAK,EAAE,QAAQ,KAAG,QAErC,CAAC;AAEF,eAAO,MAAM,GAAG,GAAI,KAAK,EAAE,QAAQ,KAAG,KAErC,CAAC"}
 
-//// https://sokra.github.io/source-map-visualization#base64,aW1wb3J0IHsgU29tZVR5cGUgfSBmcm9tICJzb21lLWRlcCI7DQpleHBvcnQgZGVjbGFyZSBjb25zdCBmb286ICh0aGluZzogU29tZVR5cGUpID0+IFNvbWVUeXBlOw0KZXhwb3J0IGRlY2xhcmUgY29uc3QgYmFyOiAodGhpbmc6IFNvbWVUeXBlKSA9PiBPdGhlcjsNCi8vIyBzb3VyY2VNYXBwaW5nVVJMPWluZGV4LmQudHMubWFw,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiaW5kZXguZC50cyIsInNvdXJjZVJvb3QiOiIiLCJzb3VyY2VzIjpbImluZGV4LnRzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUFBLE9BQU8sRUFBRSxRQUFRLEVBQUUsTUFBTSxVQUFVLENBQUM7QUFFcEMsZUFBTyxNQUFNLEdBQUcsVUFBVyxRQUFRLEtBQUcsUUFFckMsQ0FBQztBQUVGLGVBQU8sTUFBTSxHQUFHLFVBQVcsUUFBUSxVQUVsQyxDQUFDIn0=,aW1wb3J0IHsgU29tZVR5cGUgfSBmcm9tICJzb21lLWRlcCI7CgpleHBvcnQgY29uc3QgZm9vID0gKHRoaW5nOiBTb21lVHlwZSk6IFNvbWVUeXBlID0+IHsKICByZXR1cm4gdGhpbmc7Cn07CgpleHBvcnQgY29uc3QgYmFyID0gKHRoaW5nOiBTb21lVHlwZSk6IE90aGVyID0+IHsKICByZXR1cm4gdGhpbmcuYXJnOwp9Ow==
+//// https://sokra.github.io/source-map-visualization#base64,aW1wb3J0IHsgU29tZVR5cGUgfSBmcm9tICJzb21lLWRlcCI7DQpleHBvcnQgZGVjbGFyZSBjb25zdCBmb286ICh0aGluZzogU29tZVR5cGUpID0+IFNvbWVUeXBlOw0KZXhwb3J0IGRlY2xhcmUgY29uc3QgYmFyOiAodGhpbmc6IFNvbWVUeXBlKSA9PiBPdGhlcjsNCi8vIyBzb3VyY2VNYXBwaW5nVVJMPWluZGV4LmQudHMubWFw,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiaW5kZXguZC50cyIsInNvdXJjZVJvb3QiOiIiLCJzb3VyY2VzIjpbImluZGV4LnRzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUFBLE9BQU8sRUFBRSxRQUFRLEVBQUUsTUFBTSxVQUFVLENBQUM7QUFFcEMsZUFBTyxNQUFNLEdBQUcsR0FBSSxLQUFLLEVBQUUsUUFBUSxLQUFHLFFBRXJDLENBQUM7QUFFRixlQUFPLE1BQU0sR0FBRyxHQUFJLEtBQUssRUFBRSxRQUFRLEtBQUcsS0FFckMsQ0FBQyJ9,aW1wb3J0IHsgU29tZVR5cGUgfSBmcm9tICJzb21lLWRlcCI7CgpleHBvcnQgY29uc3QgZm9vID0gKHRoaW5nOiBTb21lVHlwZSk6IFNvbWVUeXBlID0+IHsKICByZXR1cm4gdGhpbmc7Cn07CgpleHBvcnQgY29uc3QgYmFyID0gKHRoaW5nOiBTb21lVHlwZSk6IE90aGVyID0+IHsKICByZXR1cm4gdGhpbmcuYXJnOwp9Ow==
 
